db/start.go

```diff
@@ -241,6 +245,46 @@ func ensureSteampipeServer() error {
 	return nil
 }
 
+func (c *Client) setServiceSearchPath() {
+	// set the search_path to the available foreign schemas or the one set by the user
+	// we need to do this here, since postgres resets the search_path on every load.
+	var schemas []string
+
+	if viper.IsSet("database.search-path") {
+		fmt.Println("got key")
+		schemas = viper.GetStringSlice("database.search-path")
+	} else {
+		schemas = c.schemaMetadata.GetSchemas()
+		// sort the schema names
+		sort.Strings(schemas)
+	}
+
+	// set this before the `public` schema gets added
```

Comment:
should we set c.schemaMetadata.SearchPath when setting the service search path?

Reply:
should be in setClientSearchPath surely?

Reply:
Till the `setClientSearchPath` actually gets called and sets the search path, this is the valid search path -  so setting it!

Reply:
I think it's needlessly confusing though.  The client will never _use_ this search path